e2e/rook-test-framework/clients/k8s_rook_file_system.go

```diff
@@ -0,0 +1,140 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+)
+
+type k8sRookFileSystem struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	fileSystemCreate    = []string{"rook", "filesystem", "create", "--name", "NAME"}
+	fileSystemDelete    = []string{"rook", "filesystem", "delete", "--name", "NAME"}
+	fileSystemList      = []string{"rook", "filesystem", "ls"}
+	wideDataToFilePod   = []string{"sh", "-c", "WRITE_DATA_CMD"}
```

Comment:
is this supposed to be *write* instead of wide?  if so, i think the same thing affects other files in this package.